docs/src/index.jsx

```diff
@@ -11,6 +11,12 @@ import ReactDOM from 'react-dom';
 import { appReduxStore } from './global/configureReduxStore';
 import routes from './routes';
 
+/**
+ * `theme` is being pushed into the window shape for documentation purposes. This gives developers,
```

Comment:
Nitpick: I'd say '... into the global \`window\` object for ...' rather than '... into the window shape for ...'.

Reply:
Fixed.

Reply:
Update in 170fcf6 looks good.  Resolving.